Libraries/Components/ActivityIndicator/ActivityIndicator.js

```diff
@@ -183,7 +184,8 @@ ActivityIndicatorWithRef.displayName = 'ActivityIndicator';
  * and run Flow. */
 ActivityIndicatorWithRef.defaultProps = {
   animating: true,
-  color: Platform.OS === 'ios' ? GRAY : null,
+  color:
+    Platform.OS === 'ios' ? GRAY : PlatformColor('?attr/colorControlActivated'),
```

Comment:
Hi @jayshah123, can you add a check and ensure that `colorControlActivated` platform color is only used on Android? 

It is the usual practice in RN repository (ex. [TextInput](https://github.com/facebook/react-native/blob/master/Libraries/Components/TextInput/TextInput.js#L45)), because there are few out-of-tree platforms and the current code will cause an issue for them.

Reply:
Yeah sure. Will update this PR.

Reply:
Updated.

Reply:
@jayshah123 I think that `default` should be a `null` to retain behavior before the Android platform fix addition. Currently this could break the out-of-tree default indicator color if they do not override the default value (if they leverage the similar behavior on theirs platform like Android on RN < 0.62, when passing `null` results in fallback to platform default color).

Reply:
Makes sense, will default to null to retain behavior.
Was thinking in the direction that some visual indicator would be better than null.
Will update the PR with your suggestions.